app/styles/global.scss

```diff
@@ -21,18 +21,28 @@ $danger: $rojo;
 $light: $anti-flash-white;
 $dark: $night;
 
+// .app-navbar {
+//   background-color: rgba(244, 245, 246, 0.4);
+//   z-index: 10000;
+//   backdrop-filter: blur(10px);
+// }
+
+// https://isotropic.co/tool/hex-color-to-css-filter/
+.app-brand-img {
+  filter: sepia(10%) contrast(130%);
+}
```

Comment:
ah yes, this is a hack to recolor the VZ logo. would be good to repaint it to match our `dark` color